HACKING.rst

```diff
@@ -57,11 +60,24 @@ Do these things once
 * Read through the cloud-init `Code Review Process`_, so you understand
   how your changes will end up in cloud-init's codebase.
 
+* Submit your first cloud-init pull request, adding yourself to the
+  in-repository list that we use to track CLA signatures:
+  ``tools/.github-cla-signers``
```

Comment:
nit: could link to the actual file here and on line 71, but obviously not necessary.